### PR TITLE
Add Flathub button and some Linux-specific info to README.me

### PR DESCRIPTION
**Pull request by Danik1601**
Added:

- Flathub button
- Flatpak installation instruction
- Flatpak configuration information
- AppImage configuration information

Changed:

- Fixed Markdown text for better compatibility according to the [guidelines](https://www.markdownguide.org/)
- Fixed HTML code for slightly better readability
- AppImage button properties
- ArchLinux button properties
- Centered Linux installation buttons
- Removed temporary Flatpak instructions

This pull request should remain in draft state until application release on Flathub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,25 +28,45 @@
...
 1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)
 
 2. Download the `.AppImage` file.
 
 3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.
 
-You can also download the flatpak version and double click the flatpak file to install via Store but wait for the full release, we'll be pushing it to Flathub for easy access.
+*Please note:*
+- *On some distributions Flatpak must be [pre-configured manually](https://flatpak.org/setup).*
+- *The AppImage needs to be [made executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80) after download.*
```
Comment (aaronliu0130) [inline README.md:69]: Should these notes be in the individual sections instead? Like the first one under flatpak and the second one under AppImage? Also, I don't think we should link the titles, as these are all notable enough and no other heading is linked.

File: README.md [hunk 2]
```diff
@@ -137,14 +161,13 @@ Upscayl uses Real-ESRGAN-ncnn-vulkan binaries to upscale images. More models and
...
-<img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
+  <img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
 <a/>
 
 # Credits ❤
 
-- Real-ESRGAN for their wonderful research work.\
+- Real-ESRGAN for their wonderful research work\.  
 [Real-ESRGAN: Copyright (c) 2021, Xintao Wang](https://github.com/xinntao/Real-ESRGAN/)
-
```
Comment (aaronliu0130) [inline README.md:147]: Is the backslash even needed now that the extra newline is gone?

**Review by aaronliu0130 — COMMENTED** [review]
Overall, looks pretty good!

File: README.md (hunk 2 above)
Comment (Danik1601) [reply README.md]: No it is not, you are right. Double space will do the job perfectly. [Done](https://github.com/Danik1601/upscayl/commit/b6b93377f2ea5d32ac3020e5fb6cbb4bfe683647)

File: README.md (hunk 1 above)
Comment (Danik1601) [reply README.md]: [Done](https://github.com/Danik1601/upscayl/commit/c38162f86f66a43c067a30b72a1a1ade57888391)

File: README.md (hunk 2 above)
Comment (Danik1601) [reply README.md]: By the way, can we just insert the link into `Real-ESRGAN` and remove the whole `Real-ESRGAN: Copyright (c) 2021, Xintao Wang` line altogether?

Comment (aaronliu0130) [reply README.md]: Maybe. You'll have to ask @NayamAmarshe though.

**Comment by Danik1601** [comment]
Should the buttons to Linux installations (Flathub, etc.) be bigger? They are tiny

**Comment by Danik1601** [comment]
How about this?

File: README.md [hunk 3]
```diff
@@ -34,34 +36,46 @@ This means that Linux users receive pre-release builds earlier but Upscayl itsel
...
+*Please note:*
+- *On some distributions Flatpak must be [pre-configured manually](https://flatpak.org/setup).*
+
+#### AppImage
 
 1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)
 
 2. Download the `.AppImage` file.
 
 3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.
```
Comment (NayamAmarshe) [inline README.md:76]:
```suggestion
Upscayl should be available on the Software Store of most Linux operating systems. You can download the Snap or [Flatpak](https://flatpak.org/setup) version, if your distro's Store App supports them.

#### Manual Install

1. Go to [releases section](https://github.com/upscayl/upscayl/releases/latest)

2. Download the `upscayl-x.x.x-linux.AppImage` file.

3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.

You can also choose to install using other formats like RPM (Fedora), DEB (Debian/Ubuntu based), and ZIP (Any Linux OS).
```

File: README.md [hunk 4]
```diff
@@ -9,19 +9,21 @@
...
 
 <a href="https://t.me/LinuxChats">
-<img src="https://user-images.githubusercontent.com/25067102/215301815-b9922b27-4e65-435a-aa9d-239da0ac61b4.png" width="200px" />
+  <img src="https://user-images.githubusercontent.com/25067102/215301815-b9922b27-4e65-435a-aa9d-239da0ac61b4.png" width="200px" />
 </a>
 
 # 🆙 Upscayl 
+
 #### Free and Open Source AI Image Upscaler
+
```
Comment (NayamAmarshe) [inline README.md:26]:
```suggestion
Upscayl lets you enlarge and enhance low-resolution images using advanced AI algorithms. It's almost like magic! 🎩🪄
```

File: README.md [hunk 5]
```diff
@@ -34,34 +36,46 @@ This means that Linux users receive pre-release builds earlier but Upscayl itsel
...
 
 1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)
 
 2. Download the `.AppImage` file.
 
 3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.
 
-You can also download the flatpak version and double click the flatpak file to install via Store but wait for the full release, we'll be pushing it to Flathub for easy access.
+*Please note:*
+- *The AppImage needs to be [made executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80) after download.*
```
Comment (NayamAmarshe) [inline README.md:78]:
```suggestion
```

This isn't needed because the instructions already mention how to make it an executable.

**Review by NayamAmarshe — CHANGES_REQUESTED** [review]
Just a few changes :)

File: README.md (hunk 3 above)
Comment (aaronliu0130) [reply README.md:76]: I don't think an AppImage is a "Manual Install", which sounds like compiling it.

File: README.md (hunk 4 above)
Comment (aaronliu0130) [reply README.md:26]: I think this should be committed to master instead of relying onthis PR

File: README.md (hunk 3 above)
Comment (NayamAmarshe) [reply README.md:76]: > I don't think an AppImage is a "Manual Install", which sounds like compiling it.

Good point

File: README.md (hunk 4 above)
Comment (NayamAmarshe) [reply README.md:26]: Hmm, it's just one line change and I didn't want to cause any merge conflicts, so I thought it'd be better to just suggest it here.

**Comment by Danik1601** [comment]
Merged them all. How about https://github.com/upscayl/upscayl/pull/301#discussion_r1189099594 ?

File: README.md (hunk 2 above)
Comment (NayamAmarshe) [reply README.md]: I think we should give proper credits, the current one is fine :)

**Comment by Danik1601** [comment]
> Should we use “Portable Method” or “AppImage”?

@NayamAmarshe suggested that it should be “Portable Method” in https://github.com/upscayl/upscayl/pull/301#discussion_r1191806478. So “Portable Method” it is, I guess.

**Comment by aaronliu0130** [comment]
Why is AppImage the only one called portable when flatpak also is?

**Comment by NayamAmarshe** [comment]
> Why is AppImage the only one called portable when flatpak also is?

Flatpak is not portable.

**Comment by NayamAmarshe** [comment]
LGTM! Thank you for the nice work!

**Pull request merged**